docs/src/pages/landing-page/Index.vue

```diff
@@ -52,6 +52,7 @@
         <why-quasar-card
           v-for="({ icon, title, body, btnLabel, btnLink }, whyQuasarCardIndex) in whyQuasar"
           :key="whyQuasarCardIndex"
+          :dark="!$q.dark.isActive"
```

Comment:
This shouldn't be there, the landing page is always in dark mode, these cards shouldn't change

Reply:
Um, that is exactly the problem. The landing page is not in dark mode. In fact, everything is working opposite to the way it should, when in dark mode. I did this as a quick fix. Putting it into proper dark mode would be a good bit more work. I'll see what I can do to revert things to work properly in dark mode.

Scott

Reply:
Ok, if I understood correctly this happens because the landing page hasn't been done to follow a "light/dark" scheme, so by default is now considered "light" since we're now using Dark plugin
Shouldn't this be a fixed value then? Or just be excluded by the whole "light/dark" thing?
The contrast is done on purpose, so it shouldn't matter if it's in dark or light mode